backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/DocumentationUnitController.java

```diff
@@ -585,8 +585,27 @@ public ResponseEntity<Void> publishDocumentationUnit(
       portalPublicationService.publishDocumentationUnitWithChangelog(uuid, user);
       return ResponseEntity.ok().build();
     } catch (DocumentationUnitNotExistsException e) {
-      log.error("Error handing over documentation unit '{}' to portal", uuid, e);
-      return ResponseEntity.internalServerError().build();
+      log.error("Could not find documentation unit with id '{}' to publish to portal", uuid, e);
+      return ResponseEntity.notFound().build();
+    }
+  }
+
+  /**
+   * Withdraws the documentation unit from the portal.
+   *
+   * @param uuid UUID of the documentation unit
+   */
+  @PutMapping(value = "/{uuid}/withdraw")
+  @PreAuthorize("@userHasWriteAccess.apply(#uuid)")
+  public ResponseEntity<Void> withdrawDocumentationUnit(
+      @PathVariable UUID uuid, @AuthenticationPrincipal OidcUser oidcUser) {
+    User user = userService.getUser(oidcUser);
+    try {
+      portalPublicationService.withdrawDocumentationUnitWithChangelog(uuid, user);
+      return ResponseEntity.ok().build();
+    } catch (DocumentationUnitNotExistsException e) {
+      log.error("Could not find documentation unit with id '{}' to withdraw from portal", uuid, e);
```

Comment:
```suggestion
      log.atError().setMessage("Could not find documentation unit to withdraw from portal").setException(e).setKeyValue("id", id).log();
```
Syntax stimmt nicht, ist einfach so konzeptionell aus dem Kopf. ⤴️ 

Mit structured logging wäre es besser, wenn wir dynamische Inhalte nicht in die Message packen, sondern als key value pair schreiben. Damit kann man sie viel leichter automatisiert auswerten und auch einfach nach der statischen Message filtern.